rep-I0005.rst

```diff
@@ -0,0 +1,164 @@
+::
+
+   REP: I0005
+   Title: Cartesian Motion Extensions for the Simple Message Protocol
+   Authord: Fred Proctor <proctor@nist.gov> and Tre Harrison <william.harrison@nist.gov>
+   Status: Active
+   Type: Process
```

Comment:
I'm not sure, but if I understand [REP-001](http://www.ros.org/reps/rep-0001.html) correctly, I think this should be a *Standards Track* REP:

> A **Standards Track** REP describes a new feature or implementation for ROS.